sympy/logic/boolalg.py

```diff
@@ -1869,6 +1869,12 @@ def _rem_redundancy(l1, terms):
 
     return essential
 
+def _binary_from_list_or_number(lon, bits):
+        def tobin(n, bits):
```

Comment:
You can use/modify `ibin` from iterables.

Reply:
Thanks!